src/transformers/testing_utils.py

```diff
@@ -240,6 +241,19 @@ def require_torchaudio(test_case):
         return test_case
 
 
+def require_torchvision(test_case):
+    """
+    Decorator marking a test that requires torchvision.
+
+    These tests are skipped when torchvision isn't installed.
+
+    """
+    if not is_torchvision_available:
```

Comment:
Seen with @sgugger, this is missing `()`:
```suggestion
    if not is_torchvision_available():
```